### PR TITLE
LPS-159864 Auto extend session doesn't work because session.timeout is not correctly read from web.xml configuration

### PR DESCRIPTION
**Pull request by jorgediaz-lr**
https://issues.liferay.com/browse/LPS-159864

Regression caused by [LPS-135438](https://issues.liferay.com/browse/LPS-135438) - Bootstrap portal web from shielded container

There are no tests of extend session functionality, they will be implemented in https://issues.liferay.com/browse/LPS-146546

I have split the `_checkWebSettings(String)` in two because:
  - I18nServlet.setLanguageIds(root); is reading the values from shielded-container-web.xml
  - The timeout information is stored in web.xml

For more information see @dantewang message https://github.com/liferay-core-infra/liferay-portal/pull/1285#discussion_r938545280

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/internal/servlet/MainServlet.java [hunk 1]
```diff
@@ -341,11 +341,16 @@ public void init() throws ServletException {
 		}
 
 		try {
-			String xml = StreamUtil.toString(
+			String shieldedContainerWebXml = StreamUtil.toString(
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portal/internal/servlet/MainServlet.java:344]: @jorgediaz-lr Please inline variables that are only used once unless SF tells you not to:
- `shieldedContainerWebXml`
- `webXml`
- `root` in `_checkShieldedContainerWebXml()` method

**Review by dantewang — CHANGES_REQUESTED** [review]
@jorgediaz-lr Please see my comment and modify your commit and force push. No need to trigger CI again. Thanks!

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
ci:forward

**Pull request closed without merging**